_includes/people/chen.daniel.html

```diff
@@ -1,8 +1,14 @@
 <div class="media">
   <img class="media-object pull-left" src="{{page.root}}/img/people/chen.daniel.jpg" alt="Daniel Chen" />
   <div id="chen.daniel" class="media-body">
-    <span class="person">Daniel Chen</span>
-    completed his Masters in Public Health in the Department of Epidemiology at Columbia University Mailman School of Public Health.
-    He is hoping to combine scientific computing and data science with epidemiology to answer questions about population health.
+    <span class="person"><a href="http://chendaniely.github.io>Daniel</a> <a href='http://twitter.com/chendaniely'>Chen</a></span>
```

Comment:
Please list your twitter ID separately in parens after your name `(<a href="...">@...</a>)`